sentry-android-timber/src/test/java/io/sentry/android/timber/SentryTimberIntegrationTest.kt

```diff
@@ -0,0 +1,55 @@
+package io.sentry.android.timber
+
+import com.nhaarman.mockitokotlin2.mock
+import io.sentry.core.IHub
+import io.sentry.core.SentryLevel
+import io.sentry.core.SentryOptions
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import timber.log.Timber
+
+class SentryTimberIntegrationTest {
+
+    private class Fixture {
+        val hub = mock<IHub>()
+        val options = SentryOptions()
+
+        fun getSut(minLevel: SentryLevel = SentryLevel.ERROR): SentryTimberIntegration {
+            return SentryTimberIntegration(minLevel)
+        }
+    }
+    private val fixture = Fixture()
+
+    @BeforeTest
+    fun beforeTest() {
+        Timber.uprootAll()
+    }
+
+    @Test
+    fun `Integrations plants a tree into Timber on register`() {
+        val sut = fixture.getSut()
+        sut.register(fixture.hub, fixture.options)
+
+        assertEquals(1, Timber.treeCount())
```

Comment:
`SentryTimberTree` already asserts that

Reply:
Yes, but this assert only checks if there is 1 tree. It could be any tree. 
The current tests don't fail 
* if we change `SentryTimberIntegration.tree` to `Timber.DebugTree.
* if we change the init of the tree to `tree = SentryTimberTree(hub, SentryLevel.ERROR)`
* if we initialize our own hub in `register`

We can keep this test, but I would say we could add one or two to make sure the `SentryTimberTree` is properly planted into Timber and we can do this for example by checking that `Timber.d(...)` calls the `hub`. We could also add one test to make sure the `minLevel` is properly passed down to `SentryTimberTree`.

Reply:
I can't `verify` which tree is planted as `Timber.plant` is a static call or I'd need an adapter for that, it's not worth IMO.
Adding a test for checking the minLevels and calling hub to be sure it planted "any" tree that calls our Hub.
also did a `instanceof` test, guess it's better now, thanks